README.md

```diff
@@ -49,15 +49,7 @@ Godel: Austrian logician and mathematician http://en.wikipedia.org/wiki/Kurt_G%C
   sim_robot:=false robot_ip:=[robot ip]
   ```
 
-- Bring up Kinect2 sensor, if available:
```

Comment:
Probably a separate issue, but should we document supported hardware?

Also, we have a few references to the kinetic sensor that we've re-purposed for the ensenso (e.g. the static publisher from kinetic2_move_frame). If we are officially dropping support for the Kinect sensor, we should rename/remove those references.